### PR TITLE
feat: Create donation page

### PR DESCRIPTION
**Pull request by Neel0851**

### CODE REVIEW

File: src/Pages/Donation/index.tsx [hunk 1]
```diff
@@ -1,75 +1,54 @@
...
-    // TODO: Add Cashfree API integration here to process the payment
-
-    console.log("Submitting form data:", formData);
-  };
+  return (
+    <TableRow>
+      <TableCell>{label}</TableCell>
+      <TableCell>{value}</TableCell>
+      <TableCell>
+        <CopyButton copied={isCopied} onClick={() => copyToClipboard(value)}>
```
Comment (vaidehi27) [inline src/Pages/Donation/index.tsx]:
```suggestion
        <CopyButton copied={isCopied} onClick={copyToClipboard}>
```

File: src/Pages/Donation/index.tsx [hunk 2]
```diff
@@ -1,75 +1,54 @@
...
+import { accountInfo, bhagavadGita } from "./constant";
+
+const mapBHAGAVAD_GITA = bhagavadGita.map((paragraph) => (
+  <Paragraph key={paragraph}>{paragraph}</Paragraph>
+));
+
+const CopiedTableRow = ({ label, value }: { label: string; value: string }) => {
+  const [isCopied, setIsCopied] = useState(false);
+
+  const copyToClipboard = (text: string) => {
```
Comment (vaidehi27) [inline src/Pages/Donation/index.tsx]:
```suggestion
  const copyToClipboard = () => {
```

File: src/Pages/Donation/index.tsx [hunk 3]
```diff
@@ -1,75 +1,54 @@
...
+    navigator.clipboard.writeText(text);
```
Comment (vaidehi27) [inline src/Pages/Donation/index.tsx]:
```suggestion
    navigator.clipboard.writeText(value);
```

**Review by vaidehi27 — APPROVED** [review]

File: src/Components/Dropdown/constant.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+export const Dropdownitems = [
```
Comment (vaidehi27) [inline src/Components/Dropdown/constant.ts]:
```suggestion
export const dropdownItems = [
```

File: src/Pages/Donation/index.tsx [hunk 5]
```diff
@@ -1,75 +1,26 @@
...
-    // TODO: Add Cashfree API integration here to process the payment
-
-    console.log("Submitting form data:", formData);
-  };
+import { Wrapper, Title, Table, Paragraph, ImageDiv, Image } from "./style";
+import { accountInfo, bhagavadGita } from "./constant";
+import { CopiedTableRow } from "./CopiedTableRow";
+import TitleBase from "../../assets/images/Titlebase.png";
 
+const DonationPage = () => {
```
Comment (vaidehi27) [inline src/Pages/Donation/index.tsx]:
```suggestion
export const DonationPage = () => {
```

File: src/Pages/Donation/style.ts [hunk 6]
```diff
@@ -1,44 +1,103 @@
...
   }
 `;
+
+export const TableStyle1 = styled.div`
+  margin: 0px 50px 10px;
+`;
+
+export const TableStyle2 = styled.div`
+  margin: 0px 50px 10px;
+`;
```
Comment (vaidehi27) [inline src/Pages/Donation/style.ts]:
```suggestion
export const TableStyle = styled.div`
  margin: 0px 50px 10px;
`;
```

File: src/Pages/Donation/style.ts [hunk 7]
```diff
@@ -1,44 +1,103 @@
...
+  margin: 2rem;
 `;
 
-export const InputGroup = styled.div`
+export const Table = styled.table`
+  border-collapse: collapse;
+  width: 100%;
+`;
+
+export const TableData = styled.td`
```
Comment (vaidehi27) [inline src/Pages/Donation/style.ts]:
```suggestion
export const TableColumn = styled.td`
```

File: src/Pages/Donation/style.ts [hunk 8]
```diff
@@ -1,44 +1,103 @@
...
+`;
+
+export const Title = styled.h1`
+  font-size: 3rem;
+  font-weight: bold;
+  font-family: "Times New Roman", Times, serif;
+  margin-top: 2rem;
+`;
+
+export const ImageDiv = styled.div`
```
Comment (vaidehi27) [inline src/Pages/Donation/style.ts]:
```suggestion
export const ImageWrapper = styled.div`
```

File: src/Pages/Donation/style.ts [hunk 9]
```diff
@@ -1,44 +1,103 @@
...
+    width: 100%;
+    max-width: 100%;
+  }
+  @media screen and (min-width: 769px) {
+    width: 60%;
+    max-width: 50%;
+  }
+`;
+
+export const TableWrapper = styled.table`
```
Comment (vaidehi27) [inline src/Pages/Donation/style.ts]:
```suggestion
export const Table = styled.table`
```

File: src/Pages/Donation/constant.ts [hunk 10]
```diff
@@ -0,0 +1,51 @@
+export const bhagavadGita = [
+  "In this endeavour of offering service to Lord Krishna, you get eternal credit, and a little service offer to Lord Krishna can protect one from the most dangerous type of fear. --Bhagavad Gita 2.4",
+  "While Krishna was going to the fruit vendor very hastily, most of the grains He was holding fell. Nonetheless, the fruit vendor filled Krishna’s hands with fruits, and her fruit basket was immediately filled with jewels and gold. --Srimad Bhagavatam 10.11.11",
+  "Please donate and attain the supreme blessings of Supreme Lord Krishna.",
+];
+
+export const accountInfo1 = [
```
Comment (vaidehi27) [inline src/Pages/Donation/constant.ts]:
```suggestion
export const yesBankAccountInfo = [
```

File: src/Pages/Donation/constant.ts [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    label: "Account Number",
+    value: "019494600000047",
+  },
+  {
+    label: "IFSC Code",
+    value: "YESB 0000194",
+  },
+];
+
+export const accountInfo2 = [
```
Comment (vaidehi27) [inline src/Pages/Donation/constant.ts]:
```suggestion
export const iciciBankAccountInfo = [
```

File: src/Pages/Donation/style.ts [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+export const TableCell = styled.td`
+  border: 1px solid #ddd;
+  padding: 0.5rem;
+  text-align: left;
+`;
+
+export const CopyButton = styled.button<{ copied: boolean }>`
+  margin-left: 1rem;
+  padding: 0.5rem;
+  background-color: ${({ copied }) => (copied ? "green" : "#4caf50")};
```
Comment (vaidehi27) [inline src/Pages/Donation/style.ts]: use color code

File: src/Pages/Donation/constant.ts (hunk 11 above)
Comment (Priyansh1825) [reply src/Pages/Donation/constant.ts]: changes done

File: src/Pages/Donation/style.ts (hunk 12 above)
Comment (Priyansh1825) [reply src/Pages/Donation/style.ts]: changes done

File: src/Pages/Donation/constant.ts (hunk 10 above)
Comment (Priyansh1825) [reply src/Pages/Donation/constant.ts]: changes done

**Review by vaidehi27 — APPROVED** [review]

File: src/Pages/Donation/index.tsx [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+  return (
+    <Wrapper>
+      <Title>Donation</Title>
+      <ImageWrapper>
+        <Image src={titleBaseImg} />
+      </ImageWrapper>
+      {bhagavadGita.map((paragraph) => (
+        <Paragraph key={paragraph}>{paragraph}</Paragraph>
+      ))}
+      <>
```
Comment (vaidehi27) [inline src/Pages/Donation/index.tsx]: Is this empty tag needed?

File: src/Pages/Donation/style.ts (hunk 7 above)
Comment (Priyansh1825) [reply src/Pages/Donation/style.ts]: changes done

**Review by vaidehi27 — APPROVED** [review]

**Pull request merged**